src/Core/Repositories/SqlServer/OrganizationUserRepository.cs

```diff
@@ -76,6 +76,20 @@ public async Task<int> GetCountByOrganizationAsync(Guid organizationId, string e
             }
         }
 
+        public async Task<IEnumerable<string>> GetExistingByOrganizationAsync(Guid organizationId, IEnumerable<string> emails,
```

Comment:
Existing feels like an unfortunate name in this context since in some ways it could be reasoned that `existing == registered`. `GetAllByOrganizationAsync` perhaps?

Reply:
Hmm. I added existing to try and specify that we're filtering out emails that don't exist on any org user.

I'm open to changes, but I don't think `GetAllByOrganizationAsync` indicates that. Perhaps `SelectKnownEmails` or `FilterNonExistingEmails` or something?

Reply:
Ah I missed that we only return the emails and not the full users. Existing does make a bit of sense in that context but perhaps `SelectKnownEmails`.

Reply:
👍 yeah, I think that's clearer too